tests/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+addopts = -ra --showlocals --tb=native
+sensitive_url = https://settings-writer.prod.mozaws.net/v1/admin
```

Comment:
Since we use `setup.cfg` as the kitchen sink of configuration, should we add this config there instead of this new `pytest.ini` file?

Reply:
https://github.com/mozilla-services/kinto-dist/pull/1953/commits/fc71b694dde9fc57aab13067763dc6dba3658932#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52R20-R26